### PR TITLE
"Fixes Bot toss"

### PR DESCRIPTION
**Pull request by alphadose**

### CODE REVIEW

File: scripts/toss.coffee [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 
 module.exports = (robot) ->
-  robot.hear /toss/i, (res) ->
+  robot.respond /toss/i, (res) ->
```
Comment (asutoshpalai) [inline scripts/toss.coffee]: Are you sure that this won't work for `bot I don't want a toss`?  or for `bot info alphatoss` ?

Comment (alphadose) [inline scripts/toss.coffee]: mangal said so :|

Comment (asutoshpalai) [inline scripts/toss.coffee]: That's not an excuse!

This is your work and not mangal's. You are responsible for the code.

Comment (alphadose) [inline scripts/toss.coffee]: Yes because other bot scripts also use robot.respond instead of robot.hear :3

**Comment by karandesai-96** [comment]
Already done.

**Pull request closed without merging**